command/quota_apply_test.go

```diff
@@ -97,3 +99,44 @@ func TestQuotaApplyCommand_Good_JSON(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Len(t, quotas, 1)
 }
+
+func TestQuotaApplyNetwork(t *testing.T) {
+	t.Parallel()
+
+	type example struct {
+		hcl string
+		q   *api.QuotaSpec
+		e   string
+	}
+
+	mbits := 20
+
+	cases := []example{{
```

Comment:
nitpick - for table driven tests, it's common to use anonymous structs without naming it [1], and it's the pattern elsewhere here [2] (not the formatting of braces and descriptive naming as well).

[1] https://github.com/golang/go/wiki/TableDrivenTests
[2] https://github.com/hashicorp/nomad/blob/v0.10.0/scheduler/reconcile_test.go#L1076-L1093